ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1. What is the DOM?
+it is the document object model
+2. What is an event?
+an event reacts to an action when something else procs it.
+3. What is an event listener?
+an action that reacts when something procs it.
+4. What is a Constructor?
+something inside a class that lists/constructs its key/value pairs
```

Comment:
you're right about the events and event listeners getting proc'd on the page (I have never in my entire life heard someone use "proc" outside of league of legends lmao) but your answer number 4 is a little bit off. The constructor is a function that applies to every single instance of the object (so it'll apply each time you call `new ObjectName`)

does that make sense? yours was just a little bit off from what I was specifically looking for